znai-server/src/main/java/com/twosigma/znai/server/support/OnSupportFinishedServerHandler.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twosigma.znai.server.support;
+
+public class OnSupportFinishedServerHandler {
```

Comment:
feels like an extra layer we can get rid off

Reply:
Other ones can be removed as well.
But I will only change the new class.

Reply:
Can you elaborate on other ones?

Reply:
I don't see any benefit to have indirections for remove and upload.
For example, all of methods in DocumentationUploadHandler can be static, since the class doesn't have to keep states.

Reply:
Are you talking about`OnUploadFinishedServerHandler`? It is an extension point that different companies may need to overload differently. 

I have a feeling we may be talking about different things.